index.bs

```diff
@@ -132,6 +130,8 @@ This default audio session is represented as an {{AudioSession}} object that is
   };
 </pre>
 
+The [=audio session=]'s state may change, which will automatically [=update the state=] of its {{AudioSession}} object.
```

Comment:
I suppose we will have another PR to add `onstatechange` event and also mention it here?

Reply:
Yes, there is an algorithm in https://github.com/w3c/audio-session/pull/23 that fires the state change event.